models/search.go

```diff
@@ -118,9 +122,10 @@ type QueryContainer struct {
 
 // SearchRawQuery represents the format of search query
 type SearchRawQuery struct {
-	Offset int `json:"from,omitempty"`
-	Limit  int `json:"size,omitempty"`
-	Query  struct {
+	Offset   int    `json:"from,omitempty"`
+	Limit    int    `json:"size,omitempty"`
+	SortTime string `json:"sort_time,omitempty"`
```

Comment:
Are we accepting query parameter with an underscore? Usually, we follow lower camel case. Right?